server/controller/adminController.js

```diff
@@ -1,90 +1,87 @@
 const bcrypt = require('bcrypt');
-const db = require('../db');
-const jwtGenerator = require('../utils/jwtGenerator');
+const pool = require('../models/db');
+// const jwtGenerator = require('../utils/jwtGenerator');
 const queries = require('../queries/adminQuery');
 const jwt = require('jsonwebtoken')
+require('../models/userModel')();
+const { createToken, verifyToken } = require('../utils/jwtGenerator')
+
+
 
- 
 // ADMIN AND EMPLOYEE REGISTER
 
-const register = async(req, res)=>{
-    try {
-        const { firstname,lastname,email,password,gender,jobRole, department,isAdmin, address } = req.body;
-db.pool.connect((err, client, done) => {
+class UserController {
 
-    client.query(queries.createNewUser, [firstname, lastname, email, bcryptPassword,gender,jobRole,department, isAdmin,address], (error, data) => {
-        done();
-         const token = jwtGenerator(data.rows[0].id);
+  static async register(req, res) {
+        try {
+          const { firstname,lastname,email,password,gender,jobRole, department,isAdmin, address } = req.body;
```

Comment:
for javascript please use camel calse firstname ==> firstName like what you did for 'jobRole'

Reply:
Ok sir